src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.module.css

```diff
@@ -0,0 +1,63 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 30px 1fr 30px;
+    grid-template-rows: auto;
+  }
+}
+
+.carouselWrapper {
+  grid-column: 2;
+}
+
+@media (--tablet-and-down) {
+  .carouselWrapper {
+    column-gap: 10px;
+    grid-column: 1/ span 13;
+    grid-column: 1 / span 13;
+    grid-template-columns: repeat(13, minmax(0, 1fr));
+    padding: 0;
+  }
+}
+
+.slideQuote {
+  background: var(--color-gray-300);
+  /*Height 100% is to make all slides the same height */
+  height: 100%;
+  margin: 0;
+  padding: 60px 50px;
```

Comment:
You'll need to override this in a media query for mobile, since the padding is on 30px on mobile.

Reply:
👍 Added media query for mobile override padding with 30px.